tests/src/io/openliberty/tools/eclipse/test/it/LibertyPluginSWTBotGradleTest.java

```diff
@@ -12,7 +12,12 @@
 *******************************************************************************/
 package io.openliberty.tools.eclipse.test.it;
 
```

Comment:
High-level question:  is this all just refactoring/reformatting? Did any test logic get changed?

Reply:
check initialize() and similar updates caused by the fact that the structure holding the projects has changed. The rest is just eclipse formatting that got rid of spaces, tabs, etc.